buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -45,11 +45,17 @@ class AirbytePythonPlugin implements Plugin<Project> {
         project.task('installReqs', type: PythonTask) {
             module = "pip"
             command = "install -r requirements.txt"
+            inputs.file('requirements.txt')
+            outputs.file('build/installedreqs.txt')
+            outputs.cacheIf { true }
```

Comment:
what's the implication of this? it just always caches its output?

Reply:
This tells it that it's cacheable, but it still needs to re-run if the inputs change.